content/learn/book/migration-guides/0.7-0.8/_index.md

```diff
@@ -0,0 +1,399 @@
+
+`Timer::times_finished has` been renamed to `Timer::times_finished_this_tick` for clarity.
+
+### [Change default Image FilterMode to Linear](https://github.com/bevyengine/bevy/pull/4465)
+
+```rs
+// 0.7
+
+//TODO
+
+// 0.8
+
+// TODO
+```
+
+### [Remove .system()](https://github.com/bevyengine/bevy/pull/4499)
+
+You can no longer use `.system()`. It was deprecated in 0.7.0. You can just remove the method call.
+
+If you needed this for tests purposes, you can use `bevy_ecs::system::assert_is_system` instead.
+
+### [Change gamepad.rs tuples to normal structs](https://github.com/bevyengine/bevy/pull/4519)
+
+The `Gamepad`, `GamepadButton`, `GamepadAxis`, `GamepadEvent` and `GamepadEventRaw` types are now normal structs instead of tuple structs and have a new() function. To migrate change every instantiation to use the `new()` function instead and use the appropriate field names instead of .0 and .1.
+
+### [Remove EntityMut::get_unchecked](https://github.com/bevyengine/bevy/pull/4547)
+
+Replace calls to `EntityMut::get_unchecked` with calls to `EntityMut::get`.
+
+### [Replace ReadOnlyFetch with ReadOnlyWorldQuery](https://github.com/bevyengine/bevy/pull/4626)
+
+The trait ReadOnlyFetch has been replaced with ReadOnlyWorldQuery along with the WorldQueryGats::ReadOnlyFetch assoc type which has been replaced with <WorldQuery::ReadOnly as WorldQueryGats>::Fetch
+
+The trait `ReadOnlyFetch` has been replaced with `ReadOnlyWorldQuery` along with the `WorldQueryGats::ReadOnlyFetch` assoc type which has been replaced with `<WorldQuery::ReadOnly as WorldQueryGats>::Fetch`
+
+* Any where clauses such as `QueryFetch<Q>: ReadOnlyFetch` should be replaced with `Q: ReadOnlyWorldQuery`.
+* Any custom world query impls should implement `ReadOnlyWorldQuery` insead of `ReadOnlyFetch`
+
+Functions `update_component_access` and `update_archetype_component_access` have been moved from the `FetchState` trait to `WorldQuery`
+
+* Any callers should now call `Q::update_component_access(state` instead of `state.update_component_access` (and `update_archetype_component_access` respectively)
+* Any custom world query impls should move the functions from the `FetchState` impl to `WorldQuery` impl
+
+`WorldQuery` has been made an `unsafe trait`, `FetchState` has been made a safe `trait`. (I think this is how it should have always been, but regardless this is _definitely_ necessary now that the two functions have been moved to `WorldQuery`)
+
+* If you have a custom `FetchState` impl make it a normal `impl` instead of `unsafe impl`
+* If you have a custom `WorldQuery` impl make it an `unsafe impl`, if your code was sound before it is going to still be sound
+
+### [Fix unsoundness with Or/AnyOf/Option component access'](https://github.com/bevyengine/bevy/pull/4659)
+
+<!-- TODO make this a bit friendlier -->
+
+If you are now getting query conflicts from `Or`/`AnyOf`/`Option` rip to you and ur welcome for it now being caught.
```

Comment:
```suggestion
Query conflicts from `Or`/`AnyOf`/`Option` have been made fixed, and made stricter to avoid UB.
If you have new query conflicts due to this you must refactor your systems; consider using `ParamSet`.
```

Reply:
Did you mean `have been fixed`? Also, I think I'd prefer not using acronyms for UB since this could be read by beginners.